wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -1253,6 +1253,48 @@
 		}
 	};
 
+	Wicket.Ajax.ChanneledAjax = Wicket.Class.create();
+
+	Wicket.Ajax.ChanneledAjax.prototype = {
+
+		initialize: function(channelName, channelType, channelManager, ajaxTransport) {
+			this.name = channelName;
+			this.type = channelType;
+			this.channel = this.name + "|" + this.type;
+			this.channelManager = channelManager;
+			this.transport = ajaxTransport? ajaxTransport : jQuery.ajax;
+		},
+
+		ajax: function(url, options) {
+			if (typeof url === "object") {
+				options = url;
+				url = undefined;
```

Comment:
This looks wrong. IMO if `url` is Object then:
 * if `options` is Object too then merge them
 * if `options` is undefined then: `options = url; url = options.u || options.url`

There is no unit test for `url` being Object.